tasks/actorstate/task.go

```diff
@@ -151,8 +151,8 @@ func (t *Task) startActorStateExtraction(ctx context.Context, current, executed
 							Head:    ac.Actor.Head,
 							Address: addr,
 						}
-						stop := metrics.Timer(ctx, metrics.StateExtractionDuration)
-						defer stop()
+						// stop := metrics.Timer(ctx, metrics.StateExtractionDuration)
+						// defer stop()
```

Comment:
What is the motivation for this the removal of this? If we _do_ mean to remove it, let's opt to delete the lines instead of commenting it out. And if this is the only location this metric is instrumented, let's delete the metric too.

Reply:
I think we figure out a better way is to drop unwanted metrics in prometheus. I'll revert this.